kratos/solving_strategies/strategies/butcher_tableau.h

```diff
@@ -241,15 +243,15 @@ class ButcherTableauForwardEuler : public ButcherTableau<ButcherTableauForwardEu
 
     static const BaseType::VectorType GenerateWeights()
     {
-        VectorType B;
-        B(0) = 1.0;
+        VectorType B(1);
+        B[0] = 1.0;
```

Comment:
```suggestion
        VectorType B(1, 1.0);
```

Reply:
Same comment, I found it clearer in the other way (even though it is more verbose). Note that Butcher tableau might be a bit confusing for newbies.